styles/details.scss

```diff
@@ -0,0 +1,54 @@
+.details {
```

Comment:
I don't see any media queries, and it shluld be adjusted to smaller devices as well

Reply:
Media queries added